cmd/cogito/main.go

```diff
@@ -32,14 +32,21 @@ func run(in io.Reader, out io.Writer, logOut io.Writer, args []string) error {
 	})
 	log.Info(cogito.BuildInfo())
 
+	ghAPI := os.Getenv("COGITO_GITHUB_API")
```

Comment:
This is undocumented on purpose and makes sense only when running tests, to pass the mock http server.
It is done here in main.go because it allows to deeply test almost all the code, without having to spawn a subprocess.

See how it is used in `TestRunPutSuccess` in cmd/cogito/main_test.go